main.js

```diff
@@ -233,3 +251,10 @@ function showView(view) {
     }
   }
 }
+
+function addToCart(cart) {
```

Comment:
How about `renderCartCount`? This function doesn't actually add anything to a cart.